docs/reference/setup/install/docker.asciidoc

```diff
@@ -546,12 +533,16 @@ To manually set the heap size in production, bind mount a <<set-jvm-options,JVM
 options>> file under `/usr/share/elasticsearch/config/jvm.options.d` that
 includes your desired <<set-jvm-heap-size,heap size>> settings.
 
-For testing, you can also manually set the heap size using the `ES_JAVA_OPTS`
-environment variable. For example, to use 16GB, specify `-e
-ES_JAVA_OPTS="-Xms16g -Xmx16g"` with `docker run`. The `ES_JAVA_OPTS` variable
-overrides all other JVM options. We do not recommend using `ES_JAVA_OPTS` in
-production. The `docker-compose.yml` file above sets the heap size to 512MB.
+For testing, you can manually set the heap size using the `ES_JAVA_OPTS`
+environment variable. Example:
+
+[source,sh]
+----
+docker run -e ES_JAVA_OPTS="-Xms1g -Xmx1g" --name es01 --net elastic -p 9200:9200 -it -m 2GB {docker-image}
```

Comment:
@henningandersen Can you give me a reality check on this setup (1GB heap for a 2GB container)?